streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -3447,4 +3449,51 @@ void process(final ProcessorSupplier<? super K, ? super V> processorSupplier,
     void process(final ProcessorSupplier<? super K, ? super V> processorSupplier,
                  final Named named,
                  final String... stateStoreNames);
+
+    /**
+     * Convert this stream to a {@link KTable}.
+     * <p>
+     * Note that this is a logical operation and only changes the "interpretation" of the stream, i.e., each record of
+     * this stream is no longer treated as an updated record (cf. {@link KStream} vs {@code KTable}).
```

Comment:
`no longer treated as an updated record` -> c&p error; it was a "fact/event" and is re-interpreted as update now.